### ISSUE TITLE
Why is Oliver showing up as credited for an Albanian translation?

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/website#396]
@oliver-phet it seems you are being credited for the Albanian translation of Acid Base Solutions

![capture](https://cloud.githubusercontent.com/assets/4051174/13549229/90d3e7bc-e2be-11e5-9124-834aa3bb9e1f.PNG)

Any insight as to why that might be?

**Comment by oliver-phet** [comment]
I think this was a test submission that was supposed to go to phet-dev - is there a way for us to easily delete this translation? @jbphet @mattpen

**Comment by jbphet** [comment]
@mattpen and I can work together during core hours today to get this removed.

**Comment by jbphet** [comment]
I just looked at the translated file in babel and the translated version of the sim, and it looks fairly legit, meaning that there are a lot of translated words that don't look like random entries.  I'm therefore not sure that this was really a "test submission" as @oliver-phet says above.  @oliver-phet - are you sure that we should just delete this?

**Comment by jbphet** [comment]
After a little more examination, the user ID in the translated Albanian file does indeed match @oliver-phet's.  This is kind of mysterious.  Was this done with Google translate or something?  Or supplied to us and then submitted by @oliver-phet?  I'd like to make sure that we understand what's going on before just deleting this.

**Comment by oliver-phet** [comment]
I was testing an HTML5 translation of a sim that already has a Java translation - it pulled the strings in from the Java sim.

**Comment by oliver-phet** [comment]
@ariel-phet Assigning this back to you. @jbphet and I discussed and this was just a resubmission of an Albanian Java translation and most of the strings are correct. @jbphet wondered if we could just leave it?

**Comment by ariel-phet** [comment]
@oliver-phet I am fine leaving it, but can credit be given to the original translator in that case?

**Comment by oliver-phet** [comment]
That would be a question for @jbphet. I have the original translator's information, but I'm not sure if we have a procedure for inserting/rebuilding the translation with that info. @jbphet can you comment?

**Comment by jbphet** [comment]
If he has a user ID we could probably just modify the strings file(s) in babel, rebuild, and work with the web site guys to republish the same version.

**Comment by oliver-phet** [comment]
The translator does have a user ID. I'll send an email with that info.

**Comment by jbphet** [comment]
I manually updated the file in babel with the correct ID for the Albanian translator and have pulled it on Figaro.  I'm not sure where the web site gets the translation credit information, but I don't think it's directly from the published simulations because I examined the source code of a sim and didn't see the translator ID in there.  The website is probably pulling the data from another table that must ultimately pull it from babel.  I'm assigning this to @mattpen to do whatever needs to be done - if anything - on the website side to update the translation credits.
